hplsql/src/main/java/org/apache/hive/hplsql/Exec.java

```diff
@@ -1028,9 +1028,13 @@ void cleanup() {
    * Output information about unhandled exceptions
    */
   public void printExceptions() {
+    List<Signal> userDefinedSignals = new ArrayList<>();
```

Comment:
Is it allowed to add the same Signal more than one times?
Does the order of `Signals` matter?

Reply:
> Is it allowed to add the same Signal more than one times?
As per code, same signal is not getting added more than once.

> Does the order of Signals matter?
Here to store the signals Stack data structure is getting used so might be the order matters thats why pushing the userdefined signals in the same order which they were present in Stack before processing other signals.